pkgs/by-name/me/mediasynclite/package.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gtk3
+, glib
+, gsettings-desktop-schemas
+, pkg-config
+, curl
+, openssl
+, jansson
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mediasynclite";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "iBroadcastMediaServices";
+    repo = "MediaSyncLiteLinux";
+    rev = version;
+    hash = "sha256-mS2e6zZutSEAr5gCS58t0Nzg8dCJgn/yoFMmbgnIM2I=";
+    };
+
+  buildInputs = [
+   curl
+   openssl.dev
+   jansson
+  ];
+
+  nativeBuildInputs = [
+   gtk3
+   glib
```

Comment:
I would inuitively move them to buildInputs rather but we can clarify this by setting strictDeps to true

Reply:
I've added strictDeps: [earlier](https://github.com/NixOS/nixpkgs/pull/294716#discussion_r1554569226) you asked me to move them to `nativeBuildInputs`

Reply:
That comment was only about line 28, gsettings-desktop-schemas, not the entire list.